src/js/structs/Pod.js

```diff
@@ -138,6 +138,10 @@ module.exports = class Pod extends Service {
     return this._spec;
   }
 
+  getStatusString() {
+    return (this.get('status') || '').toUpperCase();
```

Comment:
1. Do we really expect to get different status strings form the backend?
2. Let's add a proper constant (constant file) if we need one